samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj

```diff
@@ -35,12 +35,18 @@
     <Reference Include="Avro, Version=0.9.0.0, Culture=neutral, PublicKeyToken=b1248d7eeb9d0814, processorArchitecture=MSIL">
       <HintPath>..\packages\Apache.Avro.1.7.7.2\lib\Avro.dll</HintPath>
     </Reference>
+    <Reference Include="Azure.Core">
+      <HintPath>C:\Users\v-wenjyu\.nuget\packages\azure.core\1.8.1\lib\netstandard2.0\Azure.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Azure.Storage.Blobs">
+      <HintPath>C:\Users\v-wenjyu\.nuget\packages\azure.storage.blobs\12.8.1\lib\netstandard2.0\Azure.Storage.Blobs.dll</HintPath>
```

Comment:
The HintPath should not have hard coded values pointing to personal machine

Reply:
Sorry, it was my mistake!  I used the `Reference Manager` to add reference directly , and forgot to modify it.